pandas/core/strings.py

```diff
@@ -45,6 +45,15 @@ def str_cat(arr, others=None, sep=None, na_rep=None):
 
     Examples
     --------
+    When ``na_rep`` is `None` (default behavior), NaN value(s)
+    in the Series propagate and return value will be NaN.
+
+    >>> Series(['a','b',np.nan,'c']).str.cat(sep=' ')
```

Comment:
I think better to actually ignore nans (as indicated in the issue), rather than just document